docs/wiki/Upgrading.md

```diff
@@ -1,37 +1,67 @@
-# Upgrade Process
+# Updating Eventum
 
 One of our objectives is to make upgrading from an earlier release as
 painless as possible, and we provide scripts that should bring your
 existing Eventum installation up-to-date.
 
+Eventum supports sequential upgrade path using released minor versions.
+
+To explain more clearly: you may skip all patch versions and upgrade to the
+next minor version.
+
+This table lists upgrade path from SOURCE to TARGET version, the third column
+shows PHP version required for the update.
+
+| SOURCE | TARGET | PHP |
+|--------|--------|-----|
+| 2.2   | 2.3   | 5.1 |
+| 2.3.x | 2.4.0 | 5.1 |
+| 2.4.x | 3.0.0 | 5.1 |
+| 3.0.x | 3.1.0 | 5.3 |
+| 3.1.x | 3.2.0 | 5.5 |
+| 3.2.x | 3.3.0 | 5.6 |
+| 3.3.x | 3.4.0 | 5.6 |
+| 3.4.x | 3.5.0 | 5.6 |
+| 3.5.x | 3.6.0 | 7.1 |
+| 3.6.x | 3.7.0 | 7.1 |
+| 3.7.x | 3.8.0 | 7.1 |
+| 3.8.x | 3.9.0 | 7.2 |
+
+## UTF-8 is required
+
 Please note that if your database encoding is not UTF-8, you may encounter various bugs:
 
 -   PDO Driver is [unusable](https://github.com/eventum/eventum/pull/167)
 -   Issue History entries [get corrupted](https://gitter.im/eventum/eventum?at=58225f1d45c9e3eb4314b58c) (JSON requires UTF-8 encoding)
 
 See 2.2 upgrade instructions how to convert database to UTF-8.
 
-## IMPORTANT
+## Upgrade overview
 
 When upgrading to a new version of Eventum, please follow these instructions:
 
 1.  Check the [requirements](Prerequisites.md) for the version
-1.  Backup your copy of Eventum - files and data.
-2.  Extract your new Eventum copy over your existing folder structure
-3.  Run the upgrade scripts described in below
+1.  Backup your copy of Eventum - files and data
+1.  Rename the current Eventum folder out of the way (`eventum.old`)
+1.  Extract new Eventum version to existing installation path (`eventum`)
+1.  Restore config and workflow files from the previous version
+1.  Run the upgrade script
+
+This way of installing will get rid of files that got removed from newer Eventum version.
```

Comment:
It is hard to read. This might sound better: "This way of installing clears all obsolete files from Eventum installation path".

Reply:
Why it's hard to read? People don't understand such complicated words as "obsolete" and apparently don't bother to consult a dictionary either.

Here's a real-life example:
- https://github.com/request/request/issues/3142